core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -301,7 +305,16 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       // - final_workflow_outputs_dir = submissions/final-outputs
       // - final_workflow_outputs_mode = "copy".
 
+      useCromwellGcpBatchBackend: Boolean = currentSettings
+        .collectFirst { case setting: UseCromwellGcpBatchBackendSetting =>
+          setting
+        }
+        .exists(_.config.enabled)
+      cromwellSubmissionBackend =
+        if (useCromwellGcpBatchBackend) gcpBatchBackend else highSecurityNetworkCromwellBackend
+
       executionServiceWorkflowOptions = ExecutionServiceWorkflowOptions(
+        submission.submissionRoot,
         submission.submissionRoot,
```

Comment:
there's a good comment on `ExecutionServiceWorkflowOptions` that explains why we're passing the same value as the first two arguments … perhaps add a code comment here too? I was awfully confused when I first read this code, until I drilled down into `ExecutionServiceWorkflowOptions`

Reply:
I added a clarifying comment, thanks!